app/core/store/modules/products.js

```diff
@@ -29,11 +29,10 @@ export default {
     loadProducts ({ commit }) {
       commit('toggleLoadingProducts')
 
-      return new Products()
-        .fetch()
-        .done(products => commit('addProducts', { products }))
-        .fail(() => noty({ text: 'Failed to load product pricing', type: 'error' }))
-        .always(() => commit('toggleLoadingProducts'))
+      const productsRequest = new Products().fetch()
```

Comment:
This change is for clarity, right? I like this better, and I assume the `return` here was implicit from coffescript->js at some point.

Reply:
The Backbone fetch option didn't chain all the error handling together throwing a `res.catch` error in the console. This change correctly hooks up the callbacks without the error.

Fixed since I was in the area.